plutus-pab-client/default.nix

```diff
@@ -9,9 +9,19 @@ let
   '';
 
   # For dev usage
-  generate-purescript = pkgs.writeShellScript "plutus-pab-generate-purescript" ''
+  generate-purescript = pkgs.writeShellScriptBin "plutus-pab-generate-purs" ''
     rm -rf ./generated
-    ${server-invoker}/bin/plutus-pab psgenerator generated
+    cp ${haskell.packages.plutus-pab.src}/plutus-pab.yaml.sample plutus-pab.yaml
```

Comment:
I would say that this should only happen if plutus-pab.yaml doesn't exist already, otherwise if someone has their own customizations for dev they will keep being written over every time they run nix-shell or lorri does etc.

Reply:
Good point, that would be very annoying I guess. I will make that change. Thanks @shmish111